lib/configurators/aws/aws.go

```diff
@@ -327,58 +333,51 @@ func (c *ConfiguratorConfig) CheckAndSetDefaults() error {
 	if !c.Flags.Manual {
 		var err error
 
-		if c.AWSSession == nil {
-			c.AWSSession, err = awssession.NewSessionWithOptions(awssession.Options{
-				SharedConfigState: awssession.SharedConfigEnable,
-				Config: aws.Config{
-					UseFIPSEndpoint: useFIPSEndpoint,
-				},
-			})
-			if err != nil {
-				return trace.Wrap(err)
-			}
+		awsConfig, err := config.LoadDefaultConfig(ctx,
```

Comment:
will the unit test fails if host does not have aws credentials?

Reply:
ah good point, it does not. I'll fix that.

Reply:
done https://github.com/gravitational/teleport/pull/46367/commits/24b9ab917818ba69a533d964980925132e85afa1